Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRenderer.cs

```diff
@@ -258,7 +258,7 @@ private void RegisterStarupScriptIfNeeded (BocBooleanValueRenderingContext rende
       scriptBuilder.Append ("$(this).parent().children('a').children('img').first()[0]");
       scriptBuilder.Append (", ");
       if (renderingContext.Control.ShowDescription)
-        scriptBuilder.Append ("$(this).parent().children('span').first()[0]");
+        scriptBuilder.Append ("$(this).parent().children('span').slice(1, 2)[0]");
```

Comment:
The selector was simply changed to chose the second instead of the first item, reflecting the updated elements sequence. 

It would have been possible to also use the elements ID during selection but this would result in a larger change and was therefor not targeted for this bugfix.

Reply:
What's the reason behind choosing `.slice(1, 2)[0]` instead of simply `[1]`?

Reply:
I did not realize that jquery has this option when working with the children() selector.

Reply:
@iam-mholle it's now updated to use the shorter syntax consistently.